tardis/conftest.py

```diff
@@ -75,6 +75,9 @@ def pytest_addoption(parser):
     parser.addoption("--less-packets",
                      action="store_true", default=False,
                      help="Run integration tests with less packets.")
+    parser.addoption("--py.test",dest='py.test',
+                     action="store_true", default=False,
+                     help="Run Tardis full test.")
 
 
 # -------------------------------------------------------------------------
```

Comment:
Well :) - the idea was to have a more illustrate command line argument, something like "--full-testing", or "--slow".